### ISSUE TITLE
Add a description to network types

### ISSUE THREAD

**Issue by ynbrthr** [peeringdb/peeringdb#680]
Provide a help text when editing the network type field. 

E.g. for Network Services #463: "Covers networks that offer network-centric services. Some example: DNS, NTP, Whois, RDAP, DDOS protection"

**Comment by ynbrthr** [comment]
@peeringdb/pc your vote please. The idea is to describe what we intend with those types to limit misclassifications.

**Comment by arnoldnipper** [comment]
+1

**Comment by grizz** [comment]
+1

**Comment by shane-kerr** [comment]
+1

**Comment by grizz** [comment]
@ynbrthr @peeringdb/pc 

Can we get a list of the help text for each type?

How should it be displayed?

**Comment by mcmanuss8** [comment]
+1 - assuming these strings are translatable, we should give a heads up to translators once they're added

**Comment by ynbrthr** [comment]
Alright: brought back here for wider review:

- Content: Content providers and delivery networks, cloud providers, hosting services, and, in general, other outbound-heavy networks
- NSP: wholesale transit providers
- Network services: Covers networks that offer network-centric services. e.g., DNS, NTP, Whois, RDAP, DDoS mitigation
- Government: Government networks, all levels (e.g. municipal, state, federal, ...)
- Enterprise: Enterprise networks providing connectivity to their employees, to branches, subsidiaries, ...
- Educational: Schools, universities, other training and teaching institutions 
- Non-Profit: non-profit organisations
- Cable/DSL/ISP: Networks providing last-mile and retail Internet services

comments welcome.

Still to be determined: how to display that / which space we’d have

**Comment by arnoldnipper** [comment]
+1

**Comment by grizz** [comment]
Waiting on where / how to display.

**Comment by ynbrthr** [comment]
How about a tooltip? poping-up on the right side, or above / below of the type selection box when a user hovers over a particular type? In that case, we'd display the helper text for 'Enterprise'

<img width="414" alt="image" src="https://user-images.githubusercontent.com/58094221/81407435-6432b780-9109-11ea-809b-93fd105f1579.png">

[btw, why do we have two 'not disclosed' entries there?]

**Comment by grizz** [comment]
> [btw, why do we have two 'not disclosed' entries there?]

peeringdb/django-peeringdb/issues/30

**Comment by Yo-Robinson** [comment]
@ynbrthr, I like the tooltip idea, it keeps the drop-down clean and simple. +1 from me.

**Comment by ynbrthr** [comment]
@peeringdb/pc, any issue with using a tooltip for that? Other ideas? 

@peeringdb/20c, do we still need to discuss implementation details?

**Comment by grizz** [comment]
@ynbrthr there isn't an easy way to display tooltips for each drop down item.

We could replace the dropdowns with autocomplete boxes, but that would require a fair amount of work.

**Comment by arnoldnipper** [comment]
What about having a larger tooltip box next to "Network type"?

**Comment by grizz** [comment]
I'm not sure what you mean?

I think maybe a help icon with a mouse of list of all of them would be good. We could add that for other fields as well.

**Comment by ynbrthr** [comment]
> I'm not sure what you mean?
> 
> 
> 
> I think maybe a help icon with a mouse of list of all of them would be good. We could add that for other fields as well.

So, sure, a help icon. I prefer a tooltip-like display cause you don’t have to explicitly click on something to get what those categories are for, but if the former is way easier to implement, let’s go with that.

I don’t think you want to hear my opinion for UX-related matters in any case :)

I guess I should now reach out to the translator team to let them know of what’s coming.

**Comment by ynbrthr** [comment]
I forgot route servers, and we need to add one for route collectors (#734). Here goes the full list:

- Content: Content providers and delivery networks, cloud providers, hosting services, and, in general, other outbound-heavy networks
- NSP: Wholesale transit providers
- Network services: Covers networks that offer network-centric services. e.g., DNS, NTP, Whois, RDAP, DDoS mitigation
- Government: Government networks, all levels (e.g. municipal, state, federal, ...)
- Enterprise: Enterprise networks providing connectivity to their employees, to branches, subsidiaries, ...
- Educational: Schools, universities, other training and teaching institutions
- Non-Profit: Non-profit organisations
- Cable/DSL/ISP: Networks providing last-mile and retail Internet services
- Route Server: BGP-speaking node allowing for multilateral peering relationships over the IX; routes learned from one peer are advertised to all the others
- Route Collector: BGP-speaking node whose role is to collect all routes from participants; does not advertise any route

@grizz : given that tooltips are hard to implement with a dropdown menu and that you lean toward having a help icon, is there anything else left to discuss? If not, we can move that to '4'

**Comment by as44980** [comment]
>     * Cable/DSL/ISP: Networks providing last-mile and retail Internet services

Could this network type be more succinctly described as "Access", perhaps with the list of common media types used to deliver this shifted to the right of the colon? ;)

`</pedant>`

**Comment by ynbrthr** [comment]
whatever could help folks understand that, when you mostly sell access, even if you have a wholesale BU, this is where you belong, rather than in the NSP category :)

However methinks we'd need a new ticket to change the type here

**Comment by Yo-Robinson** [comment]
+1 I'm in support of the "tooltip" idea. Users would find it helpful to
understand what the network type is about. The idea that @Yan proposed
regarding the 'help' functionaility sounds good to me.
